misc/python/materialize/mzbuild.py

```diff
@@ -302,8 +302,28 @@ def extra(self) -> str:
         return ",".join(flags)
 
 
+def generate_cargo_build_command(
+    bins: List[str],
+    examples: List[str],
+    rd: RepositoryDetails,
```

Comment:
nit: I think it would be a bit cleaner to push the `rd` extraction into this function.